sdk/storage/azdatalake/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+* Fixed an issue where customers could not capture the raw HTTP response of directory and file GetProperties operations.
```

Comment:
Could I get more info on how this is broken?  Mostly I'm wondering if this is something we need to fix in `azcore`.

Reply:
So some datalake APIs call into the blob API. GetProperties on the blob endpoint returns both blob properties and some HNS properties like owner, group, etc. Since those properties are HNS only, they arent in the BlobGetPropertiesResponse, so it cant be directly converted to or aliased as the File/DirGetPropertiesResponse. The way we currently implement this is we call blob get properties and capture the raw response (with includeRawResponse context) to grab the specific HNS property headers. But if a customer also passes this context, we effectively override the same context key resulting in the raw response to be captured internally, and the customer is left with nil as the raw response. Basically we need to be able to handle multiple context keys to prevent this overriding, one for internal use and one for customer use.

Reply:
Got it, thanks for the explanation.